### DISCUSSION TITLE
I have a question about the database region.

### DISCUSSION THREAD

**Discussion by Kir93** [planetscale/discussion#389] [Q&A] [answered]
Will the Seoul region be added?
If yes, when will it be added?

I really want to be added.  🙏

**Comment by orware** [comment, marked as answer]
Hello @Kir93,

We appreciate your suggestion for the Seoul region!

New regions for our regular Cloud plans are decided upon by our team, with new ones regularly being added over time.

I would like to mention that for our [Managed Cloud](https://planetscale.com/docs/concepts/deployment-options) offering, it provides extra flexibility to allow for other regions not supported yet by our regular Cloud offering since users utilize their own AWS/GCP accounts, but it requires an Enterprise plan.

**Reply by Kir93** [reply]
Thanks for your reply. Hope to add it soon.
